selection_sort/python/selection_sort.py

```diff
@@ -0,0 +1,20 @@
+# Function to do selection sort of given array(Arr)
+def selection_sort(Arr):
```

Comment:
Write docs for the method. Get help from other python's implementations in this repo.